.coveragerc

```
@@ -90,6 +90,8 @@ omit =
     homeassistant/components/azure_devops/sensor.py
     homeassistant/components/azure_service_bus/*
     homeassistant/components/baidu/tts.py
+    homeassistant/components/balboa/__init__.py
+    homeassistant/components/balboa/balboa_entity.py
```

```suggestion
    homeassistant/components/balboa/entity.py
```

Usually we don't prefix a file name with the namespace its already in